Vector/Views/RoomTitle/PreviewRoomTitleView.m

```diff
@@ -114,30 +121,31 @@ - (void)refreshDisplay
             }
         }
         
-        // FIXME: Display members status when it will be available
-        self.roomMembers.text = nil;
-//        if (memberCount)
-//        {
-//            if (activeCount > 1)
-//            {
-//                self.roomMembers.text = [NSString stringWithFormat:NSLocalizedStringFromTable(@"room_title_multiple_active_members", @"Vector", nil), activeCount, memberCount];
-//            }
-//            else
-//            {
-//                self.roomMembers.text = [NSString stringWithFormat:NSLocalizedStringFromTable(@"room_title_one_active_member", @"Vector", nil), activeCount, memberCount];
-//            }
-//        }
-//        else
-//        {
-//            // Should not happen
-//            self.roomMembers.text = nil;
-//        }
+        // Display members status
+        if (memberCount)
```

Comment:
The information cannot be accurate in case of private room with preview = OFF.
We cannot know all the members nor their presence